spec/requests/servers_spec.rb

```diff
@@ -110,7 +110,7 @@
       server = FactoryBot.create(:miq_server, :status => 'started')
 
       expect { post(api_server_url(nil, server), :params => gen_request(:delete)) }.to change(MiqServer, :count).by(0)
-      expect_single_action_result(:success => false, :message => 'Failed to destroy the record')
+      expect_single_action_result(:success => false, :message => 'Failed to destroy')
```

Comment:
Rails change the method here to add more record information so it's easier to use a simpler string check to satisfy this on rails 6.1 and 7.